hw/ip_templates/clkmgr/rtl/clkmgr.sv.tpl

```diff
@@ -290,6 +293,16 @@ rg_srcs = list(sorted({sig['src_name'] for sig
     // divider step down controls
     .step_down_acks_i(step_down_acks)
   );
+  % else:
+  // No bypass as there are no derived clocks
+
+  // Read inputs and tie-off outputs
+  logic unused_bypass = ^{lc_clk_byp_req_i, all_clk_byp_ack_i, io_clk_byp_ack_i};
```

Comment:
This suggests you could remove the ast *clk_byp_* logic from the AST, and remove the corresponding clkmgr ports.

Reply:
Sounds like a plan.